tools/cli.js

```diff
@@ -116,9 +124,41 @@ commandProcessor.complete = (inputs, depth) => {
 
 commandProcessor.call = (interfaceName, methodName, args, callback) => {
   if (!state.client) return callback(new Error('Not connected'));
+
+  interfaceName = tryComplete(interfaceName, Object.keys(state.api));
```

Comment:
Do we really need to complete this?

I do see a value in allowing users to type commands incompletely, that's both convenient and consistent with tools like vi. But completing method names upon Enter without requiring the user to press Tab may lead to confusing mistakes and running random commands inadvertently. I'd prefer to force users to be explicit here.